stubs/Symfony/Component/Security/Core/Authorization/Voter/Voter.stub

```diff
@@ -13,10 +13,11 @@ abstract class Voter implements VoterInterface
     /**
      * Determines if the attribute and subject are supported by this voter.
      *
-     * @param mixed $subject
+     * @phpstan-param TAttribute $attribute
```

Comment:
This is a wrong fix because it's okay to ask about anything in `supports`. The `TAttribute` is typehinted only in `voteOnAttribute` because only valid `TAttribute` can be passed there. But in `supports` you can ask about whatever.

Reply:
Fixed as per your [comment](https://github.com/phpstan/phpstan-symfony/issues/367#issuecomment-1785362022).